discretenet/problems/school_bus_scheduling/school_bus_scheduling.py

```diff
@@ -0,0 +1,185 @@
+import numpy.random as random
```

Comment:
Even though this problem doesn't strictly need it, can you add an empty `__init__.py` file next to this one for consistency

Reply:
Updated